File: python_questions.py
import random

correct_answers = 0

def python_question_1(tries):
    global correct_answers 
    print("Question ID: 0001")
    print("Programming Language: Python")

    print("How to print Hello World!")

    for i in range(tries):
        answer = input()
        if answer == 'print("Hello World!")':
            print("Success!")
            correct_answers += 1
            break
        else:
            if (tries - i) == 1:
                print("Failure!")
                print("Do you want to know the correct answer? Input yes or no.")
                truth = input()
                if truth == "Yes" or truth == "yes":
                    print("Correct Answer:")
                    print('print("Hello World!")')
            elif (tries - i) == 2:
                print("You only have 1 try left.")
            else:
                print("You only have " + str(tries - i - 1) +" tries left.")

def total_correct():
    return correct_answers

def send_questions():
    return [python_question_1]

File: javascript_work/python_questions.js
let python_question_1 = {
    language: 'Python',
    type: 'Written',
    question: 'How to print "Hello, World!" in Python?',
    answer: 'print("Hello, World!")',
};

let python_question_2 = {
    language: 'Python',
    type: 'Written',
    question: 'Write a comment in Python with this text: This is a comment',
    answer: '# This is a comment',
};

let python_question_3 = {
    language: 'Python',
    type: 'Written',
    question: 'Write a variable x with the value of 11.',
    answer: 'x = 11',
};



export function python_question_generator(){
    console.log(`Programming Language: Python`);
    return [python_question_1, python_question_2, python_question_3];
};

